docs/support.md

```diff
@@ -2,4 +2,4 @@
 
 The LoRaWAN starter kit is an open source solution, it is NOT a Microsoft
 supported solution or product. For bugs and issues with the codebase please log
-an issue in this repo.
+an issue in the [GitHub repository](https://github.com/Azure/iotedge-lorawan-starterkit/issues/395).
```

Comment:
@roel4ez this link is pointing to a specific issue, we should point to the issues section.